crates/primitives/src/lib.rs

```diff
@@ -74,3 +74,16 @@ pub mod serde_bincode_compat {
         transaction::{serde_bincode_compat as transaction, serde_bincode_compat::*},
     };
 }
+
+/// Temp helper struct for integrating [`NodePrimitives`](reth_primitives_traits::NodePrimitives).
+#[derive(Debug, Clone, Default, PartialEq, Eq)]
+pub struct AnyPrimitives;
+
+impl reth_primitives_traits::NodePrimitives for AnyPrimitives {
```

Comment:
I think it makes sense to just use `EthPrimitives` for this?

Reply:
the reason I made `AnyPrimitives`, is so that we can replace type by type inside `EthPrimitives` and `OpPrimitives`, component by component, and don't have to wait till we can replace a type on `EthPrimitives` and `OpPrimitives` across all components at the same time

Reply:
> we can replace type by type inside `EthPrimitives`

wdym? I'd expect EthPrimitives to be the same as right now (besides transaction type being changed eventually). so given that AnyPrimitives are same as EthPrimitives right now this would probably hold?

Reply:
yes but it is subject to change and not in all components at once :)